src/util/BaseRestClient.ts

```diff
@@ -92,6 +92,7 @@ export default abstract class BaseRestClient {
   constructor(
     restOptions: RestClientOptions = {},
     networkOptions: AxiosRequestConfig = {},
+    useDemoTrading: boolean = false,
```

Comment:
Instead of adding a 3rd parameter, can you mode it to a new property in RestClientOptions? This is what I did for bybit, would love for it to be consistent (same place/name/type): 
https://github.com/tiagosiebler/bybit-api/blob/master/src/util/requestUtils.ts#L16-L19

Reply:
No problem, I'll move it to `RestClientOptions` and update the prop name for consistency.